### PR TITLE
Xcode Fixes

### PR DESCRIPTION
**Pull request by erincatto**
Fix cppdialect=C++11 (Xcode 9.3.1 does not like c++0x)
Fix angle bracket include complaints for building Premake in Xcode.

### CODE REVIEW

**Comment by tdesveauxPKFX** [comment]
Out of curiosity, what is xcode complaining about with `#include <>`?

Otherwise, this looks fine to me, will still leave approval to someone more familiar with XCode.
Might also make sense to split this in two PR?

File: modules/xcode/xcode_common.lua [hunk 1]
```diff
@@ -1032,7 +1032,7 @@
 	xcode.cppLanguageStandards = {
 		["Default"] = "compiler-default",  -- explicit compiler default
 		["C++98"] = "c++98",
-		["C++11"] = "c++0x",      -- Xcode project GUI uses c++0x, but c++11 also works
+		["C++11"] = "c++11",      -- Xcode project GUI uses c++0x, but c++11 also works
```
Comment (samsinsane) [inline modules/xcode/xcode_common.lua]: This comment is no longer relevant.

**Comment by erincatto** [comment]
Here is an example error:
/premake-core/binmodules/example/main.c:1:10: 'luashim.h' file not found with <angled> include; use "quotes" instead

File: modules/xcode/xcode_common.lua (hunk 1 above)
Comment (erincatto) [reply modules/xcode/xcode_common.lua]: Done

**Pull request merged**